sshtun

```diff
@@ -180,12 +231,10 @@ while getopts ":cklhr:ps:v" opt; do
 		r)
 			if [ -n "$PROFILE" ]; then
 				run_firefox_with_profile "$PROFILE"
```

Comment:
We now have only one Firefox profile specified in configuration. Does it make sense to remove the parameter and read it from configuration?